tests/rptest/tests/datalake/mount_unmount_test.py

```diff
@@ -73,14 +73,14 @@ def __init__(self, test_context):
             redpanda=self.redpanda,
             include_query_engines=[QueryEngineType.SPARK])
 
-    def avro_stream_config(self, topic, subject, cnt=3000):
+    def avro_stream_config(self, topic, subject, cnt=3000, interval_ms=None):
```

Comment:
>Production rate reduced as otherwise RPCN produces too much data while unmounting

Hmm I thought we blocked writes while we unmounted. Are we sure we eventually actually quiesce? The tests I saw were hanging around for 12 minutes without completing -- I don't imagine unmount would take _that_ long, would it?

Reply:
Yes we do, but it takes some time since we block it.
The 12 minutes you saw was a combination of many problems, swallowed errors in particular -- see 1st commit.